src/index.js

```diff
@@ -1,11 +1,14 @@
-import * as task from './modules/populate.js';
-import * as stat from './modules/status.js';
+import * as task from './modules/status.js';
 import './style.css';
 
-let list = [];
-const listEl = document.querySelector('ul');
+let list = [
+  { description: 'Set up a new project with webpack', isCompleted: false, index: 0 },
+  { description: 'Set up a new project with webpack', isCompleted: false, index: 1 },
+  { description: 'Create an index.js file', isCompleted: false, index: 2 },
+  { description: 'Write a function to iterate over the tasks array and populate an HTML', isCompleted: false, index: 3 },
+];
```

Comment:
Please remove the dummy data and let the user add his/her todo items from scratch :)